src/os/bluestore/bluestore_types.cc

```diff
@@ -477,12 +480,12 @@ void bluestore_blob_use_tracker_t::get(
     total_bytes += length;
   } else {
     auto end = offset + length;
-    if (end / au_size >= num_au) {
+    if (end >> au_size >= num_au) {
```

Comment:
It has to be end >> **au_size_exponent**, right? If so - there is a bunch of similar issues below as well.